pkg/secrethub/repo.go

```diff
@@ -107,6 +130,45 @@ func (s repoService) ListEvents(path string, subjectTypes api.AuditSubjectTypeLi
 	return events, nil
 }
 
+// EventIterator returns an iterator that retrieves all audit events for a given repo.
+//
+// Usage:
+//  iter, err := client.Repos().EventIterator(path)
+//  if err != nil {
+//  	   // Handle error
+//  }
+//  for {
+//      event, err := iter.Next()
+//	    if err == secrethub.IteratorDone {
+//	        break
+//	    }
+//	    if err != nil {
+//	        // Handle error
```

Comment:
let's just make this say `return err` or something.

Reply:
Why? It's up to the user to decide how to deal with the error, isn't it?